### ISSUE TITLE
FR: display catalogs and nested manifests

### ISSUE THREAD

**Issue by flammable** [munkireport/munkireport-php#836]
Similar to #835, it'd be really awesome if one could view a client's nested manifests and catalogs from MunkiReport.  At the moment, I'm using tags for this, but if both were available in sortable lists, we could determine the following things in our environment directly from MunkiReport:

- Whether a machine has certain testing catalogs enabled or not
- Based on the nested manifests, whether a machine is considered a faculty/staff computer or a lab computer
- If it's a lab computer, which lab is it located in?
- Much more!

The downside to using tags is human error: we're creating manifests all the time, and I'm making changes to the Munki repository and constantly forgetting to update a client's tags.  Additionally, we've had a few machines repurposed, and they retained their tags - as one would expect - and I needed to adjust their tags based on how they're being used now.  It'd be awesome if that information updated when the client checked in with new nested manifests.

Thanks!

**Comment by nathanperkins** [comment]
Would you need to preserve the hierarchy of the nested manifests or would you be okay with comma-separated values?

If you're okay with comma-separated values, I think it wouldn't be hard to write a munki-facts script once the munki_conditions module is merged into Munki Report.

**Comment by flammable** [comment]
Thanks, @nathanperkins! I'm not too worried about the hierarchy - searchable should be plenty useful!  I'm not attached to them being sortable, to be honest.

Also, it looks like your Munki Conditions module already lists the catalogs!  For my purposes, that should be perfect.  Thanks again!

**Comment by cwhits** [comment]
Replicating `manifestutil expand-included-manifests` would be neat since that breaks down catalogs, and manifests in a nested fashion.

**Comment by gmarnin** [comment]
As a side note, if you have MWA2 running, you can access the manifest via a link on the Munki tab in the client details report. PR https://github.com/munkireport/munkireport-php/pull/523

**Comment by gmarnin** [comment]
The [manifest module](https://github.com/joncrain/manifests) does most of this
